spec/requests/profiles_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe "Profiles", type: :request do
+  describe "GET /index" do
+    pending "add some examples (or delete) #{__FILE__}"
```

Comment:
@Shieka @phkeese this prints a warning: https://github.com/hpi-swt2/bookkeeper-portal-red/actions/runs/3480448631/jobs/5820258887#step:5:35

I'll go ahead and delete the file, since we already tests for this in spec/features/users/show_spec.rb.